scripts/init-cron-nightly.sh

```diff
@@ -0,0 +1,58 @@
+H1='\033[0;31m'
+H2='\033[0;32m'
+NC='\033[0m' # No color
+
+# Variables
+build_id="14543"
+tensile_path="`pwd`"
+
+# Constants
+script_to_run="profile-tcl.sh"
```

Comment:
How would we reuse this or run multiple scripts?

Reply:
To re-use this with other script, we can extend this to accept a script as aa command line parameter. For now, this is the only script we intend to use so leaving it hard coded seems like an acceptable solution.